src/components/NavigationItems.js

```diff
@@ -16,8 +16,8 @@ const iconLibrary = {
   'Build apps': 'buildApps',
   'Automate workflows': 'automation',
   'Explore docs': 'developerDocs',
+  'Attend events': 'event',
```

Comment:
I somehow missed that this event icon for the new relic icon library got added. I like keeping that one, so lets do it!